scripts/stanncam_manager/stanncam_manager.gml

```diff
@@ -60,6 +60,33 @@ function stanncam_init(_game_w, _game_h, _resolution_w=_game_w, _resolution_h=_g
 	time_source_start(global.stanncam_time_source);
 }
 
+/// @function stanncam_destroy
+/// @description removes all stanncam references from the game, the opposite of stanncam_init
+/// @param {Bool} [application_surface_draw_enable = true]
+function stanncam_destroy(_app_surface_draw_enable = true){
+	application_surface_draw_enable(_app_surface_draw_enable);
```

Comment:
since init turns app_surface_drawing off i figured it should turn it back on by default, when removed